product_docs/docs/biganimal/release/using_cluster/06_demonstration_oracle_compatibility.mdx

```diff
@@ -238,8 +238,70 @@ In both of the examples shown here, you probably would not use the functions and
 better, more familiar or at least more widely-available equivalents provided natively by PostgreSQL (and many other databases). But by supporting them, EDB Advanced Server gives you the ability to reuse existing logic with minimal modification, allowing
 you to focus your time and expertise on solving new problems.
 
+## Try it on your own cluster: export and import
+
+If you'd like to try the examples above on your own BigAnimal cluster, you can follow the instructions below to import the example database.
+
+1. [Connect to your EDB Postgres Advanced Server cluster as edb_admin](https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/02_connecting_your_cluster/) using psql.
+
+2. Create a new database and role:
+
+    ```sql
+    CREATE DATABASE chinook;
+    \c chinook
```

Comment:
This prompts for a password. It's the `edb_admin` password used when setting up a cluster, but since I'd just set up my account on EDB, I initially thought it was that password. Maybe separate this into it's own step so that we can clarify which password is needed? (Though I did feel dumb when I realized what the command was actually doing. It's more of the copy-and-paste while trying to get through the demo quickly that causes the problem, I think.)

I also wondered if it were "password" since that's the very next line and I pasted the whole thing at once.

Reply:
I'll split this into two and make a note about the password. Could help to provide an example of connecting in step #1